### ISSUE TITLE
[Consistency] User encounters a component or pattern that isn't considered a best practice.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80105]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefits Non-Disability
**Product:** Burial Form 530
**Feature:** 21P-530 Parity Updates

## Findings details
**VA.gov Experience Standard - issue:** User encounters a component or pattern that isn't considered a best practice.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In [v2 /veteran-information/locaation-of-death/](https://staging.va.gov/burials-and-memorials-v2/application/530/veteran-information/location-of-death), conditionally revealed inputs don't follow guidance from the design system.

For the question "Where did the Veteran's death occur?" five of the six radio buttons result in new fields being revealed, and four of them reveal two fields each. Current [design system guidance](https://design.va.gov/components/form/radio-button#conditionally-revealed-fields) says no more than one new field can be revealed in an interaction.

The reasoning behind this: As a screen reader user moves through the page, there's no concept of the revealed fields being "children" of the original field, ie. nothing like the blue vertical line used to suggest that relationship to sighted users. If a screen reader user selects a radio button, begins filling in their responses to the revealed fields, then backtracks and changes their answer, they may not realize that the structure of the page will change when they head back down the page again. Limiting to one revealed field limits how much they need to parse through to understand what changed and what that relationship is.

The labels for these questions are pretty clear so I think the actual impact on a user would be relatively low --- most users will understand what happened and not be confused. But this is a best practice we're trying to enforce across VA.gov to ensure a more consistent experience.

## Recommended action
I'd recommend moving those revealed fields onto another page following the one thing per page/[ask for a single response model](https://design.va.gov/patterns/ask-users-for/a-single-response). But please reach out in Slack if there are concerns about that solution.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

### Acceptance Criteria
- [ ] Review feedback and recommended action from Platform noted above
- [ ] Audit form and propose how we can incorporate the feedback if you think we should
- [ ] Review proposal with VA enablement team for approval
- [ ] Create tickets for next steps and close this one

**Comment by fiorella-io** [comment]
I created the designs which are pending content review.
- [Selected new page](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=2309%3A22420&mode=design&t=vnxjiAu57bDm8rxG-1)
- [Other new page](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=2309%3A41206&mode=design&t=vnxjiAu57bDm8rxG-1)

**Comment by steele-lm** [comment]
CAIA has finished there review and the new designs are ready for implementation.
- [Selected new page](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=2309%3A22420&mode=design&t=vnxjiAu57bDm8rxG-1)
- [Other new page](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=2309%3A41206&mode=design&t=vnxjiAu57bDm8rxG-1)

**Comment by matt4su** [comment]
Matt to run designs by Dene first.

**Comment by matt4su** [comment]
Adding to agenda for 1/13 Product sync to determine priority

**Comment by matt4su** [comment]
Received go-ahead from Sanja to pursue as a low priority items as time allows
